website/project/views/drafts.py

```diff
@@ -0,0 +1,210 @@
+from flask import request, redirect
+import httplib as http
+from dateutil.parser import parse as parse_date
+import functools
+
+from modularodm import Q
+from modularodm.exceptions import ValidationValueError
+
+from framework import status
+from framework.mongo import database
+from framework.mongo.utils import get_or_http_error, autoload
+from framework.exceptions import HTTPError
+from framework.status import push_status_message
+
+from website.util.permissions import ADMIN
+from website.project.decorators import (
+    must_be_valid_project,
+    must_have_permission,
+    http_error_if_disk_saving_mode
+)
+from website import language
+from website.project import utils as project_utils
+from website.project.model import MetaSchema, DraftRegistration
+from website.project.metadata.utils import serialize_meta_schema, serialize_draft_registration
+from website.project.utils import serialize_node
+
+get_schema_or_fail = lambda query: get_or_http_error(MetaSchema, query)
```

Comment:
Disagree. A lambda makes it explicit that the function takes only one argument (the only use case for this function)